scripts/runtests.py

```diff
@@ -28,7 +28,7 @@ def colored(text: str, color: str | None = None, **kwargs: Any) -> str:  # type:
 _FAILED = colored("Failed", "red")
 # We're using the oldest supported version because it's the most likely to produce errors
```

Comment:
Very minor, but we could maybe tweak the comment slightly:

```suggestion
# We're using the oldest fully supported version because it's the most likely to produce errors
```